exo-setup/src/app-setup.spec.ls

```diff
@@ -0,0 +1,123 @@
+require! {
+  '../../exosphere-shared': {run-process}
+  'chai': {expect}
+  'fs-extra' : fs
+  'js-yaml' : yaml
+  'mkdirp'
+  'path'
+  'rimraf'
+}
+
+
+describe 'AppSetup', ->
+
+  describe 'app with external dependencies' (...) ->
+    @timeout 600000
+
+    before (done) ~>
+      @app-name = 'external-dependency'
```

Comment:
Let's test against a more complex app so we know that all edge cases are covered, like the todo app that's generated in `tutorial.feature`. I will send you an example.